cmd/libs/go2idl/client-gen/testoutput/testgroup/unversioned/testgroup_client.go

```diff
@@ -69,7 +69,7 @@ func setConfigDefaults(config *unversioned.Config) error {
 	if err != nil {
 		return err
 	}
-	config.Prefix = "/apis"
+	config.APIPath = "/apis"
```

Comment:
i'm not familiar with the generator - have any pointers?

On Thu, Jan 7, 2016 at 10:11 AM, Daniel Smith <notifications@github.com>
wrote:

> In
> cmd/libs/go2idl/client-gen/testoutput/testgroup/unversioned/testgroup_client.go
> <https://github.com/kubernetes/kubernetes/pull/17535#discussion_r49082111>
> :
>
> > @@ -69,7 +69,7 @@ func setConfigDefaults(config *unversioned.Config) error {
> >  	if err != nil {
> >  		return err
> >  	}
> > -	config.Prefix = "/apis"
> > +	config.APIPath = "/apis"
>
> I think you need to fix the generator, not modify its output.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17535/files#r49082111>.
>